zkstack_cli/crates/zkstack/Cargo.toml

```diff
@@ -48,7 +48,7 @@ zksync_system_constants.workspace = true
 zksync_eth_client.workspace = true
 zksync_contracts.workspace = true
 prost.workspace = true
-reqwest = "0.12.8"
+reqwest = { version = "0.12.8", features = ["blocking"] }
```

Comment:
why not workspace?

Reply:
the version wasn't coming from the workspace before my changes so I missed it, just pushed a commit that ensures there are no non-workspace versions anymore